client-app/src/pages/account/views/AccountOrders.vue

```diff
@@ -3,6 +3,37 @@
     <loading :active.sync="isLoading" :z-index="5000"></loading>
     <account-order-details-modal :order-id="selectedOrderId">
     </account-order-details-modal>
+    <div class="row">
+      <div class="col-md-6">
+        <label for="start-datepicker">{{ $t("account.orders.from") }}</label>
+        <b-form-datepicker id="start-datepicker"
```

Comment:
Start of what?
I suggest to make up a more clear name:
"orders-start-date", "orders-start-period-date", "begin-date"/"end-date" and so on..
:)

Reply:
fixed in last commit